PatientRegistrationApp/src/components/HouseholdFollowUp.vue

```diff
@@ -10,19 +10,17 @@
 		<v-row justify="center">	
 			<v-col cols="12">
 				<v-card class="d-flex flex-wrap" flat>
-					<v-card class="pa-2" flat min-width=33%>
-						<div class="font-weight-medium primary--text">Household Member #1</div>
-						<div> QR Code Placeholder</div>
-						<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].familyName}}, 
-						{{dataHouseholdPersonalInfo[0].givenName}} {{dataHouseholdPersonalInfo[0].middleName}} {{dataHouseholdPersonalInfo[0].suffix}}</span></div>
-					</v-card>
-				<template v-for="index in getNumberOfHouseholdMembers()-1">
+				<template v-for="index in getNumberOfHouseholdMembers()">
```

Comment:
Can you change the title to read "Download your QR Code" and add text below it (above the QR code itself) that says "Use this QR code to easily check-in at the site where you receive your vaccine. This QR code contains an encrypted patient identifier so we can quickly and securely identify you and retrieve your information."

![image](https://user-images.githubusercontent.com/67079030/103426884-57743f80-4b82-11eb-93e4-d56f16be659d.png)